examples/planar_vector_tile.html

```diff
@@ -0,0 +1,54 @@
+<html>
+    <head>
+        <title>Itowns - Planar + TMS + Vector Tile</title>
+
+        <style type="text/css">
+            html {
+                height: 100%;
+            }
+
+            body {
+                margin: 0;
+                overflow: hidden;
+                height: 100%;
+            }
+
+            #viewerDiv {
+                margin: auto auto;
+                width: 100%;
+                height: 100%;
```

Comment:
We could add a `example.css` file instead of copy-pasting everytime :)

Reply:
Yes !

Reply:
Did #736 about that